test/unit/org/apache/cassandra/db/guardrails/GuardrailsTest.java

```diff
@@ -250,26 +250,25 @@ public void testMinThresholdUsers() throws Throwable
         assertValid(() -> guard.guard(9, "z", false, superClientState));
     }
 
-    @Test
-    public void testDisableFlag() throws Throwable
+    public void testEnableFlag() throws Throwable
```

Comment:
This is missing the `@Test` tag, so it won't be run.

Reply:
Nice catch!

Reply:
Yeah! Thanks!